app/models/message.rb

```diff
@@ -46,7 +48,9 @@ def site_message?
   end
 
   def self.send_site_message(user_id, subject, message)
-    Message.create(recipient_id: user_id, sender_id: 0, subject: subject, message: message)
+    msg = Message.new(recipient_id: user_id, subject: subject, message: message)
+    msg.sender_id = 0
+    msg.save
```

Comment:
Can't mass-assign `sender_id` even outside a controller when using `attr_accessible` (I could get around this by making `Message` use strong params but I have inconsistently migrated to strong params and it's not necessary)